R/utils.R

```diff
@@ -61,7 +61,11 @@ get_default_cache_dir <- function() {
 #' @return `NULL`, invisibly
 #' @keywords internal
 sync_remote_file <- function(full_url, output_file, ...) {
-    if (!file.exists(output_file)) {
+    user_over <- NA
+    if (file.exists(output_file)) {
+      user_over <- tolower(readline(prompt = "Cached file already exists. Overwrite? (Y/N):"))
```

Comment:
Can you prompt with `menu` instead of `readline` please? Also check if R is running interactively: https://rlang.r-lib.org/reference/is_interactive.html.